src/twisted/conch/test/test_userauth.py

```diff
@@ -390,6 +393,329 @@ def check(ignored):
 
         return d.addCallback(check)
 
+    def test_verifyValidPrivateKey_sk_ed25519(self):
+        """
+        Test that verifying a valid sk-ssh-ed25519 public key works.
```

Comment:
Sorry for the late comment.

What I wanted to say about these docstring is to make it easier to understand what is the difference in scope between this test and `test_successfulPrivateKeyAuthentication_sk_ed25519`

I am familiar with SSH public key authentication and I can tell that this is validation for the user auth without signature... while the other test is for the userauth request with a signature... but just reading the test name and doc string, this is not very clear.